### DISCUSSION TITLE
Nuclei scan takes only about 2 seconds and does not find anything

### DISCUSSION THREAD

**Discussion by OmerYilmazlar** [projectdiscovery/nuclei#1406] [Q&A] [answered]
Hello everyone, I have a problem with nuclei-templates.

I installed nuclei and I said " nuclei -ut" for it to install the templates as well. When I type "nuclei -h", it gives me the right help menu. **However**, when I scan a website, the scan takes only about 2-3 seconds and nuclei says this;

[WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] Using Nuclei Engine 2.5.7 (latest)
[INF] Using Nuclei Templates 8.7.9 (latest)
[INF] Using Interactsh Server https://interact.sh
[INF] Templates added in last update: 2735
[INF] Templates loaded for scan: 3
[INF] Templates clustered: 3 (Reduced 2 HTTP Requests)
[INF] No results found. Better luck next time!

Every time I use nuclei, I change templates (such as vulnerabilities, cves or technologies) but this issue never changes. I give the right directory path (eg: /root/nuclei-templates/cves) and sometimes specify which exact template to use but it still gives me the same info immediately ([INF] No results found. Better luck next time!).

I know that this is impossible, because when I choose "Technologies" for example, it still says the same thing. So, even if there are no vulnerabilities in the website, at least it should tell me the technologies it utilizes. Moreover, I have nuclei-templates as [sn1per ](https://github.com/1N3/Sn1per)plugins and use them against the same website, **and it works!** I also updated the nuclei engine with "-u" flag but the issue still persists.

Does anyone knows why this might be happening? Or even if you do not know, what is your suggestion?

**Note: I removed nuclei and its templates and re-installed them both a couple of times. I even went back to some of the earliest snapshots of my virtual machine and restored them in order to check the same problem. Still, I was unable to solve the issue. I updated my Kali VM snapshots before installing nuclei and its templates.**

Thanks for your time, suggestions and answer.

**Comment by ehsandeep** [comment, marked as answer]
@OmerYilmazlar is it possible you are feeding subdomains as input instead of URLs? or can you share what you feeding as input for nuclei scan? Additionally, you can see what's going on with `-v` flag.

**Reply by OmerYilmazlar** [reply]
Hello @ehsandeep, thank you for your reply.

Here is an example of one of my scans.

By the way, I chose "-v" flag and I think I understood the problem. It says "... template was excluded" for most of the templates. So, this might be the issue. 

![image](https://user-images.githubusercontent.com/58121818/146790342-5b04afea-2904-4269-ad4f-5bc06c204318.png)

**Reply by ehsandeep** [reply]
you can read about it more here -https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion

**Reply by OmerYilmazlar** [reply]
Oh, wow. I think I solved the problem.

Now, technologies template works and it took time. So, I believe it is working this time. And the reason why vulnerabilities do not work this time might be because there is no vulnerability on that site.

![image](https://user-images.githubusercontent.com/58121818/146792571-d925841e-610b-4ba6-9be7-f3e497991145.png)
